privacyIDEA-impl/src/main/java/org/privacyidea/action/PrivacyIDEAAuthenticator.java

```diff
@@ -67,9 +82,9 @@ else if (piContext.getWebauthnSignResponse() != null && !piContext.getWebauthnSi
         }
         else if (piContext.getMode().equals("otp"))
         {
-            if (request.getParameterValues("pi_otp_input") != null)
+            if (request.getParameterValues("otp") != null)
             {
-                String otp = request.getParameterValues("pi_otp_input")[0];
+                String otp = request.getParameterValues("otp")[0];

```

Comment:
is it safe to access the first element without checking if the array(?) is empty?

Reply:
[18682b4](https://github.com/privacyidea/shibboleth-plugin/pull/50/commits/18682b4391dd678c14981a4be0dcfcf4dbedf34e)